posts/draft.md

```diff
@@ -0,0 +1,174 @@
+# A deep dive into the MyST world
```

Comment:
```suggestion
# A deep dive into MyST Markdown, Part 1: Roles and directives in Nikola
```

Reply:
I think this would make the title a bit more specific so that you could build upon it for your other parts. (feel free to re-work the wording if you like)

Reply:
> (feel free to re-work the wording if you like)

OK, let me think about this one...